migrations/2025-07-26-230802_add_default_fetch_limit/up.sql

```diff
@@ -0,0 +1,7 @@
+-- Adds an optional default fetch limit (IE fetch a certain number of posts) to local_user and local_site
+ALTER TABLE local_user
+    ADD COLUMN default_fetch_limit integer;
+
+ALTER TABLE local_site
+    ADD COLUMN default_fetch_limit integer;
```

Comment:
This should be not null with default 20, and then get rid of `FETCH_LIMIT_DEFAULT` in the code.